app/javascript/components/ActionItemSelectParticipants/index.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { Button, Dialog, DialogContent } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import ActionItemSearchParticipants from '../ActionItemSearchParticipants';
+import ActionItemDisplayParticipants from '../ActionItemDisplayParticipants';
+import styles from './styles';
+
+class ActionItemSelectParticipants extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      participants: this.props.participants,
+      selectedParticipants: [],
+      openModal: false,
+    };
+    this.addUserToState = this.addUserToState.bind(this);
+    this.removeUserFromState = this.removeUserFromState.bind(this);
+    this.addAllUsersToState = this.addAllUsersToState.bind(this);
+    this.removeAllUsersFromState = this.removeAllUsersFromState.bind(this);
+  }
+
+  // Adds selected user to state to be displayed
+  addUserToState(user) {
+    this.setState(prevState => ({
+      selectedParticipants: [...prevState.selectedParticipants, user],
+    }));
+  }
+
+  // Removes user from display
+  removeUserFromState(user) {
+    this.setState(prevState => {
+      const copy = [...prevState.selectedParticipants];
+      const index = this.state.selectedParticipants.indexOf(user);
+      copy.splice(index, 1); // Removes one element at `index` location
+      return { selectedParticipants: copy };
+    });
+  }
+
+  // Adds all users at once to be displayed
+  addAllUsersToState() {
```

Comment:
This should take in a list of participants and add only these ones since we should only be adding the participants that were visible in the search component. Ex: If someone searches for "Matthew" and then clicks the add all button, then only Matthews should be added to the participant list.

Reply:
OHHHH how interesting i didn't think about that

Reply:
i thought it would behave differently that totally makes sense ill change that up